eva-lib/src/main/java/uk/ac/ebi/eva/lib/metadata/ArchiveDBAdaptor.java

```diff
@@ -15,6 +15,9 @@
  */
 package uk.ac.ebi.eva.lib.metadata;
 
+import java.util.List;
+import uk.ac.ebi.eva.lib.models.VariantStudy;
+
```

Comment:
All the eva imports go together, separated by a new line from the previous block of imports.